samples/OpenIdConnect.AzureAdSample/project.json

```diff
@@ -8,7 +8,7 @@
     "Microsoft.Extensions.Configuration.EnvironmentVariables": "1.0.0-*",
     "Microsoft.Extensions.Configuration.UserSecrets": "1.0.0-*",
     "Microsoft.Extensions.Logging.Console": "1.0.0-*",
-    "Microsoft.IdentityModel.Clients.ActiveDirectory": "3.9.302261508-alpha"
+    "Microsoft.IdentityModel.Clients.ActiveDirectory": "3.11.305310302-alpha"
   },
   "frameworks": {
     "net451": { },
```

Comment:
3.9.302261508-alpha already worked for me, we never did find out why you had so much trouble with it.